frontend/src/hooks/pims-api/useApiDocuments.ts

```diff
@@ -27,8 +27,8 @@ export const useApiDocuments = () => {
 
       deleteDocumentRelationshipApiCall: (
```

Comment:
I realize this is from previous PR but why break the naming convention here? 
We haven't used "*ApiCall" before - any reason why this cannot be simply named **deleteDocumentRelationship** ?

Reply:
Mostly because it confuses the function that is being returned from the hook/provider vs the one that is used by the api call. I am open to suggestions about it but we should come to a stable pattern to avoid confusion in the future